### DISCUSSION TITLE
command line access to hyper:// urls?

### DISCUSSION THREAD

**Discussion by gigablast** [beakerbrowser/beaker#1786] [General]
What is the easiest way to download a hyper:// document from the linux CLI ?

obviously i can't do a wget or curl for this atm.

**Comment by pfrazee** [comment]
Yeah we don't have anything for this yet but I'm thinking about building a tool for this soon

**Comment by RangerMauve** [comment]
The dat-fetch module has a CLI which lets you load files from hyper URLs
